### PR TITLE
Add (basic) changelog

### PR DESCRIPTION
**Pull request by roidelapluie**
My proposal is to merge this, then find and add other major dates:

- Modules that joined VP (we should make monthly list)
- CoC adoption
- Initial bootstrap of Puppet-Community

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+## September 2016
+
+* New Puppet modules: puppet-boolean, puppet-sftp_jail, puppet-mrepo, puppet-php
+* New repository: ra10ke
+
+
+## August 2016
+
+* Governance document is merged (#45)
+* New mailing list: voxpupuli@groups.io
```
Comment (bastelfreak) [inline CHANGELOG.md]: should we link it to  1.freenode https://groups.io/g/voxpupuli/ ?

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+## October 2014
+
+* New Puppet module: puppet-alternatives
+* New repositories: puppetboard, pypuppetdb
+
+
+## September 2014
+
+* New Puppet modules: puppet-module-amanda, puppet-module-puppetboard,
```
Comment (bastelfreak) [inline CHANGELOG.md]: we renamed the modules to puppet-amanda and puppet-puppetboard. not sure if we want to list the original or current name.

**Comment by bastelfreak** [comment]
made two comments for things we could adjust, but don't have to. LGTM.

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,223 @@
+## October 2016
+
+* Elections ends with 5 PMC members: [@bastelfreak][bastelfreak] [@nibalizer][nibalizer] [@hunner][hunner] [@bbriggs][bbriggs]
+  [@dhoppe][dhoppe]. Election officers were [@roidelapluie][roidelapluie] and [@daenney][daenney].
```
Comment (bastelfreak) [inline CHANGELOG.md]: link for daenney doesn't work

File: CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+
+
+## August 2015
+
+* New Puppet module: [puppet-collectd][puppet-collectd]
+
+
+## July 2015
+
+* New Puppet module: [puppet-git][puppet-git]
```
Comment (bastelfreak) [inline CHANGELOG.md]: link doesn't work

File: CHANGELOG.md (hunk 3 above)
Comment (roidelapluie) [inline CHANGELOG.md]: Fixed

**Pull request merged**